Server/Server.Test/ServerWorkflowTests.cpp

```diff
@@ -0,0 +1,28 @@
+#include <catch2/catch.hpp>
+#include "TestUtility.hpp"
+
+using namespace TestUtility;
+
+TEST_CASE("Workflow startup server")
```

Comment:
Test names here and overall are not really indicative of the intent. 
You might check this (guide)[https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices] or any other google query about naming unit tests.